docker/Dockerfile_pretrained_embeddings_spacy_en_ja

```diff
@@ -0,0 +1,81 @@
+FROM python:3.7-slim as base
+
+RUN apt-get update -qq \
+ && apt-get install -y --no-install-recommends \
+    # required by psycopg2 at build and runtime
+    libpq-dev \
+     # required for health check
+    curl \
+ && apt-get autoremove -y
+
+FROM base as builder
+
+RUN apt-get update -qq && \
+  apt-get install -y --no-install-recommends \
+  build-essential \
+  wget \
+  openssh-client \
+  graphviz-dev \
+  pkg-config \
+  git-core \
+  openssl \
+  libssl-dev \
+  libffi6 \
+  libffi-dev \
+  libpng-dev
+
+# install poetry
+# keep this in sync with the version in pyproject.toml and Dockerfile
+ENV POETRY_VERSION 1.0.5
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+ENV PATH "/root/.poetry/bin:/opt/venv/bin:${PATH}"
+
+# copy files
+COPY . /build/
+COPY docker/configs/config_pretrained_embeddings_spacy_en.yml /build/config.yml
```

Comment:
I think you will need to create a different config yml file using the Japanese language model.